config/routes.rb

```diff
@@ -7,7 +7,7 @@
 
   resources :items, only: [:new, :create]
 
-  resources :recipes, only: [:index, :show, :new, :create, :destroy] do
+  resources :recipes, only: [:index, :show, :new, :create, :edit, :update, :destroy] do
```

Comment:
I think we can leave out `only:` here because we are calling all 7 CRUD actions for recipes? 😁

Reply:
Good point!